cypress/features/regression/buttonToggleGroup.feature

```diff
@@ -33,17 +33,16 @@ Feature: Button Toggle Group component
   # @ignore because of FE-1447
   # | <>                       |
 
-  @ignore
-  # ignored because the width renders into px
+  @positive
   Scenario Outline: Change Button Toggle Group input width to <width>
     When I check labelInline checkbox
       And I set inputWidth to "<width>"
     Then input width is set to "<px>"
     Examples:
-      | width | px  |
-      | 1     | 11  |
-      | 10    | 83  |
-      | 100   | 304 |
+      | width | px        |
+      | 1     | 10.75     |
+      | 10    | 107.59375 |
+      | 100   | 825.71875 |
```

Comment:
I think those kind of verification are not so precise

Reply:
Will leave these as they are for now until we can work out a solution to select only the integer part of the number. I'll log a ticket for that task.